app/scripts/modules/core/src/managed/toggleResourceManagement.ts

```diff
@@ -43,34 +63,52 @@ export const toggleResourcePause = (
   });
 };
 
+const getInterstitialBodyText = (resourceSummary: IManagedResourceSummary) => {
+  const { status } = resourceSummary;
+  let body = `
```

Comment:
could this be markdown?

Reply:
It could be markdown if we get https://github.com/spinnaker/deck/pull/7723 merged

Reply:
made it JSX